dfdatetime/golang_time.py

```diff
@@ -153,6 +153,127 @@ def _GetNumberOfSeconds(self, golang_timestamp):
 
     return number_of_seconds, nanoseconds, time_zone_offset
 
+  def _CopyNanosecondTimeFromString(self, time_string):
+    """Copies a nanosecond time from a string.
+
+    Args:
+      time_string (str): time value formatted as:
+          hh:mm:ss.#########Z
+          hh:mm:ss.#########[+-]##:##
+
+          Where # are numeric digits ranging from 0 to 9 and the seconds
```

Comment:
remove `numeric`, given you already indicate ranging from 0 to 9

Reply:
Done